cmd/bacalhau/run.go

```diff
@@ -11,6 +11,5 @@ func newRunCmd() *cobra.Command {
 		PreRun:            applyPorcelainLogLevel,
```

Comment:
should we delete this file as well?

Reply:
I deleted it in the CLI refactor: https://github.com/bacalhau-project/bacalhau/pull/2560/files#diff-0f0b9e1cadf9cb9d668a3e34dae72649ede046608eb6e38476e690994bb84949 which follows this work - would you prefer the change land in this PR?

Reply:
No need